### PR TITLE
[docs] Added "Starting browser with arguments" paragraph

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @DevExpress/testcafe-docs, @lexkazakov

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -102,6 +103,22 @@ specify a browser alias that consists of the `{browser-provider-name}` prefix an
 testcafe "saucelabs:Chrome@52.0:Windows 8.1" tests/sample-fixture.js
 ```
 
+### Starting browser with arguments
+
+If you need to pass arguments for specified browser, you should write them after browser alias and surround this construction with quotation marks:
```
Comment (lexkazakov) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: If you need to pass arguments for the specified browser, you should write them right after browser alias. Surround the browser call and its arguments with quotation marks:

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 2]
```diff
@@ -102,6 +103,22 @@ specify a browser alias that consists of the `{browser-provider-name}` prefix an
...
+
+```sh
+testcafe "chrome --start-fullscreen" tests/sample-fixture.js
+```
+
+You can also specify arguments for portable browsers. If the path to browser contains spaces, it should be surrounded with backticks:
```
Comment (lexkazakov) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: You can also specify arguments for portable browsers. If a path to a browser contains spaces, the path should be surrounded with backticks:

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 3]
```diff
@@ -102,6 +103,22 @@ specify a browser alias that consists of the `{browser-provider-name}` prefix an
...
+
+```sh
+testcafe "path:`C:\Program Files (x86)\Google\Chrome\Application\chrome.exe` --start-fullscreen" tests/sample-fixture.js
+```
+
+Specifying arguments is possible for local browsers only.
```
Comment (lexkazakov) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: You can specify arguments for local browsers only.

**Comment by georgiy-abbasov** [comment]
FPR

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 4]
```diff
@@ -102,6 +103,22 @@ specify a browser alias that consists of the `{browser-provider-name}` prefix an
 testcafe "saucelabs:Chrome@52.0:Windows 8.1" tests/sample-fixture.js
 ```
 
+### Starting browser with arguments
+
+If you need to pass arguments for the specified browser, you should write them right after browser alias. Surround the browser call and its arguments with quotation marks:
```
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: `you should` always look bad in the docs

**Comment by georgiy-abbasov** [comment]
FPR

**Comment by lexkazakov** [comment]
approved

**Pull request merged**